CRM/Mailing/Event/BAO/Subscribe.php

```diff
@@ -231,7 +231,7 @@ public function send_confirm_request($email) {
 
     $url = CRM_Utils_System::url('civicrm/mailing/confirm',
       "reset=1&cid={$this->contact_id}&sid={$this->id}&h={$this->hash}",
-      TRUE
+      TRUE, NULL, NULL, TRUE
```

Comment:
I don't think this is right - here is the fn sig

```
  public static function url(
    $path = NULL,
    $query = NULL,
    $absolute = FALSE,
    $fragment = NULL,
    $htmlize = TRUE,
    $frontend = FALSE,
    $forceBackend = FALSE
```

You are changing $htmlise too. $fragment is already NULL

Reply:
@eileenmcnaughton Hmm, I don't see how I'm changing anything by passing `NULL` for those params - the only point of passing them at all is to be able to pass `$frontend = TRUE`.

Reply:
I think the issue is that htmlize is being changed from TRUE to NULL which i think does make a material difference @christianwach

Reply:
@seamuslee001 But passing `NULL` will cause `$htmlize` to be set to its default value - i.e. `TRUE`.

Reply:
@christianwach doesn't look like it wrote this little bit of code

```
<?php

function test($var = TRUE) {
  var_dump($var);
}

test(NULL);
test();
```
running the file gives the following output 

```
NULL
bool(true)
```

Reply:
FFS, my bad. What I wrote isn't actually true is it? Over 20 years developing in PHP and... well... learn something every day :)

Reply:
well i find testing hypothesis with actual code helps ;-)

Reply:
Makes me want to open a coffee shop instead.

Reply:
Have force-pushed updated code.